concent_api/verifier/tasks.py

```diff
@@ -65,6 +79,11 @@ def blender_verification_order(
                 subtask_id,
                 VerificationResult.MISMATCH.name,
             )
+        result = str(VerificationResult.MATCH if subtask_id[-1] == 'm' else VerificationResult.MISMATCH)
```

Comment:
this check is being made twice. please refactor

Reply:
Done